torchaudio/prototype/rnnt_loss.py

```diff
@@ -98,6 +101,9 @@ def forward(
             targets (Tensor): Tensor of dimension (batch, max target length) containing targets with zero padded
             logit_lengths (Tensor): Tensor of dimension (batch) containing lengths of each sequence from encoder
             target_lengths (Tensor): Tensor of dimension (batch) containing lengths of targets for each sequence
+
+        Returns:
+            Tensor: loss with the reduction option applied.
```

Comment:
Can you add description about the shape?

Example in CE loss https://pytorch.org/docs/stable/generated/torch.nn.CrossEntropyLoss.html
>> scalar. If reduction is 'none', then the same size as the target:  ...